models/aeif_cond_beta_multisynapse.h

```diff
@@ -0,0 +1,397 @@
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef AEIF_COND_BETA_MULTISYNAPSE_H
+#define AEIF_COND_BETA_MULTISYNAPSE_H
+
+// Generated includes:
+#include "config.h"
+
+#ifdef HAVE_GSL
+
+// External includes:
+#include <gsl/gsl_errno.h>
+#include <gsl/gsl_matrix.h>
+#include <gsl/gsl_odeiv.h>
+
+// Includes from nestkernel:
+#include "archiving_node.h"
+#include "connection.h"
+#include "event.h"
+#include "nest_types.h"
+#include "ring_buffer.h"
+#include "universal_data_logger.h"
+
+/* BeginDocumentation
+ Name: aeif_cond_beta_multisynapse - Conductance based exponential
+                                      integrate-and-fire neuron model according
+                                      to Brette and Gerstner (2005) with
+                                      multiple synaptic rise time and decay
+                                      time constants, and synaptic conductance
+                                      modeled by a beta function.
+
+ Description:
+
+ aeif_cond_beta_multisynapse is an extension of
+ aeif_cond_alpha_multisynapse.  It allows an arbitrary number of synaptic
+ rise time and decay time constant. Synaptic conductance is modeled by a
+ beta function, as described by A. Roth and M.C.W. van Rossum
+ in Computational Modeling Methods for Neuroscientists, MIT Press 2013,
+ Chapter 6
+
+ The time constants are supplied by two arrays taus_rise and taus_decay for
+ the synaptic rise time and decay time, respectively.  Port numbers
+ are then automatically assigned and there range is from 1 to n.  (n
+ being the index of the last element of the tau_ex and tau_in
+ arrays).
+ During connection, the ports are selected with the property "receptor_type".
+
+ Examples:
+ % PyNEST example, of how to assign synaptic rise time and decay time
+ % to a receptor type.
+
+ neuron = nest.Create('aeif_cond_beta_multisynapse')
+ nest.SetStatus(neuron, {"V_peak": 0.0, "a": 4.0, "b":80.5})
+ nest.SetStatus(neuron, {'taus_decay':[50.0,20.0,20.0,20.0],
+                         'taus_rise':[10.0,10.0,1.0,1.0]})
+
+ spike = nest.Create('spike_generator', params = {'spike_times':
+                                                  np.array([10.0])})
+
+ voltmeter = nest.Create('voltmeter', 1, {'withgid': True})
+
+ delays=[1.0, 300.0, 500.0, 700.0]
+ w=[1.0, 1.0, 1.0, -1.0]
+ for syn in range(4):
+     nest.Connect(spike, neuron, syn_spec={'model': 'static_synapse',
+                                           'weight': w[syn],
+                                           'delay': delays[syn],
+                                           'receptor_type': 1 + syn})
+
+ nest.Connect(voltmeter, neuron)
+
+ Sends: SpikeEvent
+
+ Receives: SpikeEvent, CurrentEvent, DataLoggingRequest
+
+ Adapted by Bruno Golosio from aeif_cond_alpha_multisynapse
+ SeeAlso: aeif_cond_alpha_multisynapse
+ */
+
```

Comment:
I tried to run the example that you provide in the commet. I would add 
```
import nest 
import numpy as np

```
to make it an executable python program.

Another issue is that the resulting script fails at my machine with the following stacktrace:
```

Traceback (most recent call last):
  File "aeif_cond_alpha_multisynapse.py", line 24, in <module>
    nest.Connect(spike, neuron, syn_spec="synapse2")
  File "/home/nestml/repositories/nest_repo/bld_master_nompi/lib/python2.7/site-packages/nest/lib/hl_api_helper.py", line 230, in stack_checker_func
    return f(*args, **kwargs)
  File "/home/nestml/repositories/nest_repo/bld_master_nompi/lib/python2.7/site-packages/nest/lib/hl_api_connections.py", line 314, in Connect
    sr('Connect')
  File "/home/nestml/repositories/nest_repo/bld_master_nompi/lib/python2.7/site-packages/nest/__init__.py", line 91, in catching_sli_run
    errorname, commandname, message))
pynestkernel.NESTError: IllegalConnection in Connect_g_g_D_D: Creation of connection is not possible because:
All outgoing connections from a device must use the same synapse type.
```
Am I doing something wrong? Otherwise, could you fix the comment?